bin/sweep.js

```diff
@@ -190,11 +193,40 @@ var sweep = (function() {
                 }
                 deltas[0] /= steps;
             }
+        } else { // space is HSLuv
+            from = convert.toRgba(from);
+            to = convert.toRgba(to);
+            var fromHSLuv = hsluv.rgbToHsluv([from.r / 255, from.g / 255, from.b / 255]);
+            var toHSLuv = hsluv.rgbToHsluv([to.r / 255, to.g / 255, to.b / 255]);
+            from.h = fromHSLuv[0];
+            from.s = fromHSLuv[1] / 100;
+            from.l = fromHSLuv[2] / 100;
+            to.h = toHSLuv[0];
+            to.s = toHSLuv[1] / 100;
+            to.l = toHSLuv[2] / 100;
+            // deltas = [dH, dS, dL, dA]
```

Comment:
Since this part is copied, maybe we could keep HSL and HSLuv under the same `else` block?